manifests/base/eventbus-controller/controller-config.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: argo-events-controller-config
+data:
+  controller-config.yaml: |+
+    eventBus:
+      nats:
+        versions:
+        - version: 0.22.1
+          natsStreamingImage: nats-streaming:0.22.1
+          metricsExporterImage: natsio/prometheus-nats-exporter:0.8.0
+      jetstream:
+        # Default JetStream settings, could be overridden by EventBus JetStream specs
+        settings: |
+          # https://docs.nats.io/running-a-nats-service/configuration#jetstream
+          # Only configure "max_memory_store" or "max_file_store", do not set "store_dir" as it has been hardcoded.
+          # e.g. 1G. -1 means no limit, up to 75% of available memory
+          max_memory_store: -1
+          # e.g. 20G. -1 means no limit, Up to 1TB if available
+          max_file_store: 1TB
+        versions:
+        - version: 2.7.3
+          natsImage: nats:2.7.3
+          metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
+          startCommand: /nats-server
```

Comment:
shouldn't it be `nats-server` (without the `/`) ?

Reply:
This image is built from `scratch`, the binary is put under `/`,  so we have to use `/nats-server`.